WordPress/wordpress-playground
# Puzzle app: disable plugin onboarding

## Motivation for the change, related issues

Some puzzle pieces have onboarding flows and we had to disable them.

In a previous PR, I updated the GPT modes, but they don't seem to work.

## Implementation details

- Reverted GPT models.
- Add mu-plugins to disable onboarding flows.

## Testing Instructions (or ideally a Blueprint)

- Open [this Playground url](https://playground.wordpress.net/#{%22phpExtensionBundles%22:[%22kitchen-sink%22],%22features%22:{%22networking%22:true},%22steps%22:[{%22step%22:%22login%22},{%22step%22:%22writeFile%22,%22path%22:%22/wordpress/wp-content/mu-plugins/rewrite.php%22,%22data%22:%22%3C?php%20add_action(%20'after_setup_theme',%20function()%20{%20global%20$wp_rewrite;%20$wp_rewrite-%3Eset_permalink_structure('/%25postname%25/');%20$wp_rewrite-%3Eflush_rules();%20}%20);%22},{%22step%22:%22writeFile%22,%22path%22:%22/wordpress/wp-content/mu-plugins/disable-elementor-onboarding.php%22,%22data%22:%22%3C?php%20add_action(%20'activate_plugin',%20function()%20{%20update_option('elementor_onboarded',%20true);%20}%20);%22},{%22step%22:%22writeFile%22,%22path%22:%22/wordpress/wp-content/mu-plugins/disable-yoast-onboarding.php%22,%22data%22:%22%3C?php%20add_action(%20'wpseo_activate',%20function()%20{%20$option_array%20=%20get_option('wpseo');%20if%20(%20$option_array%20&&%20is_array(%20$option_array%20)%20)%20{%20$option_array['should_redirect_after_install_free']%20=%20false;%20update_option(%20'wpseo',%20$option_array%20);%20}%20},%201000%20);%22},{%22step%22:%22installPlugin%22,%22pluginZipFile%22:{%22resource%22:%22wordpress.org/plugins%22,%22slug%22:%22elementor%22},%22options%22:{%22activate%22:true}},{%22step%22:%22installPlugin%22,%22pluginZipFile%22:{%22resource%22:%22wordpress.org/plugins%22,%22slug%22:%22dynamic-visibility-for-elementor%22},%22options%22:{%22activate%22:true}},{%22step%22:%22writeFile%22,%22path%22:%22/wordpress/wp-content/mu-plugins/wp-seo.php%22,%22data%22:%22%3C?php%20add_action('init',%20function()%20{%20add_filter(%20'Yoast\\WP\\SEO\\should_index_indexables',%20'__return_false'%20);%20}%20);%20?%3E%22},{%22step%22:%22installPlugin%22,%22pluginZipFile%22:{%22resource%22:%22wordpress.org/plugins%22,%22slug%22:%22wordpress-seo%22},%22options%22:{%22activate%22:true}}],%22landingPage%22:%22/wp-admin/plugins.php%22}) (this is what the puzzle app would produce)
- Confirm that there are no onboarding flows loaded
